client/tests/integration/smartcontracts/create_nft_for_every_user_trigger/src/lib.rs

```diff
@@ -10,11 +8,11 @@ extern crate panic_halt;
 
 use alloc::{format, string::ToString};
 
-use iroha_wasm::{data_model::prelude::*, prelude::*};
+use iroha_trigger::prelude::*;
 
-#[iroha_wasm::main]
-fn main() {
-    iroha_wasm::info!("Executing trigger");
+#[iroha_trigger::main]
+fn main(_owner: AccountId, _event: Event) {
+    iroha_trigger::info!("Executing trigger");
```

Comment:
Minor point about language. A smart-contract is a simple "on trigger execute Executable" concept. A trigger is the mechanism that decides that something needs to run, not the thing whose `main` you're calling. 

The old naming convention was somewhat generic, I admit, but it wasn't conceptually dissonant. 

```suggestion
    iroha_hook::info!("Executing trigger");
```

or maybe

```suggestion
    iroha_smartcontract::info!("Executing trigger");
```

Personally prefer the `hook` name, because it's shorter and makes it clear that it acts like lisp hooks do, minus the dynamic typing.

Reply:
There is a different approach that one can take based on a comment below.